examples/common/imgui_ui/windows/helpers.h

```diff
@@ -0,0 +1,104 @@
+/*
+ *
+ *    Copyright (c) 2023 Project CHIP Authors
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+#include <app-common/zap-generated/attributes/Accessors.h>
```

Comment:
I would try to avoid names of helpers/support/core and similar filler name, as they end up a "grab all of everything inside".

These seem to implement a bunch of `Update*` methods, so let's figure out a name that expresses that.

Maybe `AccessorUpdaters.h` since these seem to rely on Accessor methods for get/set.